hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/BlockDeletingService.java

```diff
@@ -149,12 +151,14 @@ public BackgroundTaskQueue getTasks() {
         containerBlockInfos = builder.build();
         queue.add(containerBlockInfos);
         totalBlocks += containerBlockInfo.getNumBlocksToDelete();
+        LOG.debug("Queued- Container: {}, deleted blocks: {}",
+            containerBlockInfo.getContainerData().getContainerID(), containerBlockInfo.getNumBlocksToDelete());
       }
       metrics.incrTotalBlockChosenCount(totalBlocks);
       metrics.incrTotalContainerChosenCount(containers.size());
       if (containers.size() > 0) {
-        LOG.debug("Queued {} blocks from {} containers for deletion",
-            totalBlocks, containers.size());
+        LOG.info("In this iteration, blocksLimit was {}, elapsed time {}ms.", blocksLimitPerInterval,
```

Comment:
already have below log in methods, so this is redundant log, do not need to have this.
```
LOG.info("Chosen {}/{} blocks from {} candidate containers.",
        (originalBlockCount - blockCount), blockCount, orderedList.size());
```

Reply:
The log you have mentioned prints the blocks details, the time taken by the iteration and the limit is not showed. It would be good to have it uniformly present in all the deletion services logs.

Reply:
> The log you have mentioned prints the blocks details, the time taken by the iteration and the limit is not showed. It would be good to have it uniformly present in all the deletion services logs.

Add to the other log if required instead of adding new logs.